### PR TITLE
Improvements on gh-pages as per #34

### PR DESCRIPTION
**Pull request by luca-vercelli**
This pull request is intended for branch gh-pages, not for branch master.
Unluckily, I cannot see the final result until pages are published on gh-pages.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -20,6 +20,8 @@
     <section class="page-header">
       <h1 class="project-name">{{ site.pristine.name }}</h1>
       <h2 class="project-tagline">{{ site.description | default: site.github.project_tagline }}</h2>
+      <a href="index.html" class="btn">READE.md</a>
```
Comment (sha256) [inline _layouts/default.html:23]: These 2 links get automatically added by the next few lines.

**Comment by sha256** [comment]
You can see the final result by installing jekyll and running `./jekyll-serve.sh`

**Comment by luca-vercelli** [comment]
Thank you. I installed Jekyll and solved a pair of issues.
In the header, I have added two links to `demo.html` and `index.html` just as it were a menu.

**Comment by sha256** [comment]
@luca-vercelli, index.html is created automatically from the readme file. So they're essentially the same thing.
Also that is why we cannot delete it or rename the demo file into index.html.

**Comment by luca-vercelli** [comment]
I'm not sure of what to do yet...
The two buttons "README" and "Demos" should go together. If I remove the first, I should remove the latter too. It's meaningless a menu with 1 entry only.
If you want I can remove them both.

**Comment by sha256** [comment]
@luca-vercelli, currently there are 2 menu items:

1. View on Github (same as readme)
2. Download

You added 2 more: README and Demo

I'm saying, since View On Github is already there and it's same as README, no need to add it. So the final menu items can be:

1. Demo
2. View on Github
3. Download

**Pull request merged**